Assets/Plugins/Pixel Crushers/Dialogue System/Scripts/ConversationTimelinePlayer.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;

```

Comment:
Can we put this script in "Assets/Scripts/Dialogue/" 

Because it's our custom code, rather than the plugin's. 
We shouldn't really add any of our own stuff to the custom folder, unless it's necessary